Gemfile

```diff
@@ -76,6 +76,9 @@ gem 'yard'
 # It should not be updated without verifying that the behavior still matches JS version 4.4.2.
 gem 'zxcvbn', '0.1.7'
 
+# Lookbook must be listed after ViewComponent, so listed here out of default alphabetical order.
+gem 'lookbook', '~> 1.1.0'
```

Comment:
just to clarify my understanding... it should be fine to list it in the gemfile in any order, but must be required afterwards? And because we use `Bundler.require` in application.rb, that's how this matters? https://github.com/18F/identity-idp/blob/d54d90eb96eec6151e3356437bcda54754db2f60/config/application.rb#L16

(aside: in a past project we removed `Bundler.require` [as in this blog post](https://myronmars.to/n/dev-blog/2012/12/5-reasons-to-avoid-bundler-require), and I was a big fan)

Reply:
To be honest, I'm not entirely sure why it has to be placed after `view_component`, I'd merely followed the installation recommendations:

>Add Lookbook to your `**Gemfile**` somewhere _after_ the ViewComponent gem:

https://lookbook.build/guide/quick-start/

Reply:
I think that has to be because of `Bundler.require` but it's easier to just follow those directions for now

Reply:
👍 

I think I do rather like the article's recommendation of explicit dependencies, though I'd have to imagine that would be quite a significant lift to switch in our project.